parmed/charmm/psf.py

```diff
@@ -458,6 +458,10 @@ def load_parameters(self, parmset):
         parmset : :class:`CharmmParameterSet`
             List of all parameters
 
+        copy : bool
```

Comment:
`copy` is very vague.  Can you change this to `copy_parameters` or `copy_parameterset`?